code.user.js

```diff
@@ -3400,63 +3358,50 @@
-        function initializeMarketUI() {  
+        function initializeMarketUI() {
             $('.market_header_text').append('<progress id="see_market_progress" value="1" max="1" hidden>');
             marketProgressBar = document.getElementById('see_market_progress');
 
             // Sell orders.
-            $('.my_market_header').first().append('<div class="market_listing_buttons">' +
-
-            '<a class="item_market_action_button item_market_action_button_green select_all market_listing_button">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Select all</span>' +
-            '</a>' +
-            '<span class="separator-small"></span>' +
-
-            '<a class="item_market_action_button item_market_action_button_green select_five_from_page market_listing_button">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Select 5</span>' +
-            '</a>' +
-            '<span class="separator-small"></span>' +
-
-            '<a class="item_market_action_button item_market_action_button_green select_twentyfive_from_page market_listing_button">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Select 25</span>' +
-            '</a>' +
-            '<span class="separator-small"></span>' +
-
-            '<a class="item_market_action_button item_market_action_button_green remove_selected market_listing_button">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Remove selected</span>' +
-            '</a>' +
-            '<a class="item_market_action_button item_market_action_button_green relist_selected market_listing_button market_listing_button_right">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Relist selected</span>' +
-            '</a>' +
-            '<span class="separator-small"></span>' +
-
-            '<a class="item_market_action_button item_market_action_button_green relist_overpriced market_listing_button market_listing_button_right">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Relist overpriced</span>' +
-            '</a>' +
-            '<span class="separator-small"></span>' +
-
-            '<a class="item_market_action_button item_market_action_button_green select_overpriced market_listing_button market_listing_button_right">' +
-            '<span class="item_market_action_button_contents" style="text-transform:none">Select overpriced</span>' +
-            '</a>' +
-
-            '</div>');
+            $('.my_market_header').first().append(`<div class="market_listing_buttons">
+                <a class="item_market_action_button item_market_action_button_green select_all market_listing_button">
+                    <span class="item_market_action_button_contents">Select all</span>
+                </a>
```

Comment:
Is there a rule to put such HTML multilines on the next line? Something like this:
```suggestion
            $('.my_market_header').first().append(`
                <div class="market_listing_buttons">
                    <a class="item_market_action_button item_market_action_button_green select_all market_listing_button">
                        <span class="item_market_action_button_contents">Select all</span>
                    </a>
```

Reply:
Not that I know of.

Reply:
OK, lets stick with it as-is then.